### ISSUE TITLE
Having a parameter -p for directly passing password for jira login

### ISSUE THREAD

**Issue by shiywang** [go-jira/jira#331]
Any reasons we don't want this?

I think it would be nice to have `jira login -u xxx -p xxxxx`

**Comment by shiywang** [comment]
@coryb Is there any concerns why we don't have this flag? IF NOT, I will submit a patch.

**Comment by coryb** [comment]
No, I will not accept a PR for a password flag, it is terrible for security.  There are a number of integrations with secure password providers (gopass, keychain, etc) and also a password provider to read from stdin (ie `echo "password" | jira ...`). If none of those options work for you I would consider a PR allowing you to use an environment variable

**Comment by shiywang** [comment]
@coryb password from stdin is a workaround for me, but unfortunately, it doesn't work very well. https://github.com/go-jira/jira/issues/308
